src/pyOpenMS/converters/pandas_utils.py

```
@@ -0,0 +1,21 @@
+        pandas.DataFrame: DataFrame representation
+        
+    Raises:
+        ImportError: If pandas is not installed
+    """
+    try:
+        import pandas as pd
+    except ImportError as e:
+        raise ImportError(
+            "The 'pandas' package is required for DataFrame conversion.\n"
+            "Install with: pip install pandas"
+        ) from e
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)